README-prerequisite.md

```diff
@@ -0,0 +1,45 @@
+# Alfresco Digital Business Platform Prerequisites
+
+The Alfresco Digital Business Platform Deployment requires:
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ---------------------- |
+| Docker      | 18.09.0      | https://docs.docker.com/ |
+| Kubernetes  | 1.10.3       | https://kubernetes.io/docs/tutorials/kubernetes-basics/ |
+| Kubectl     | 1.10.3       | https://kubernetes.io/docs/tasks/tools/install-kubectl/ |
+| Helm        | 2.9.1        | https://docs.helm.sh/using_helm/#quickstart-guide |
+
+In addition, since it can be deployed to different environments, check the requirements for:
+- [KOPS deployment](#kops)
+- [EKS deployment](#eks)
+- [Docker for Desktop deployment](##docker-for-desktop---mac)
+- [Minikube deployment (officially not supported anymore)](#minikube)
```

Comment:
@jottley, what about Minikube? Should I mention this here? Or just leave the minikube readme as it is?

Reply:
@rpopa I think that because we are no longer supporting Minikube we shouldn't need to add it. It can stay in its own readme.

Reply:
Done!